gradio/components/chatbot.py

```diff
@@ -172,6 +174,7 @@ def __init__(
         likeable: bool = False,
         layout: Literal["panel", "bubble"] | None = None,
         placeholder: str | None = None,
+        examples: list[ExampleMessage] | None = None,
```

Comment:
I know we discussed what to call this, but now I'm thinking `examples` might be confusing to users since this does not follow the same behavior as examples typically do for components. What do you think about `suggestions`?